lib/services.rb

```diff
@@ -4,7 +4,11 @@ module Services
   def self.publishing_api
     @publishing_api ||= GdsApi::PublishingApiV2.new(
       Plek.new.find('publishing-api'),
-      disable_cache: true,
+      cache_ttl: 5.minutes,
```

Comment:
Ah. That's much better - will rework this. Publishing platform team are also looking at improving the request itself, so hopefully there will be less need for this soon.